pkg/skaffold/runner/deploy_test.go

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright 2019 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package runner
+
+import (
+	"bytes"
+	"context"
+	"strings"
+	"testing"
+
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/build"
+	"github.com/GoogleContainerTools/skaffold/testutil"
+	"github.com/pkg/errors"
+)
+
+func TestDeploy(t *testing.T) {
+	expectedOutput := "Waiting for deployments to stabalize"
+	var tests = []struct {
+		description string
+		testBench   *TestBench
+		shdWait     bool
+		statusCheck bool
+		shouldError bool
```

Comment:
```suggestion
		expectedError bool
```

Reply:
i will rename this to `shouldErr`  since it is shouldErr everywhere.

```
git grep 'shouldErr' | wc
    328    1179   22858

```